src/components/CSettingsItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <v-list-item
+    :prepend-icon="prependIcon"
+    :append-icon="clickable ? mdiChevronRight : undefined"
+    :ripple="clickable"
+    min-height="64"
+    class="px-md-8 px-5"
+    @click="$emit('click')"
```

Comment:
I think we should use the return of the `defineEmits` function instead:
![image](https://github.com/Crystal-Creations-GbR/crystal-components/assets/79226886/e5197800-5722-4387-8820-9c66458cdd49)
I believe it has better type safety

See https://vuejs.org/guide/components/events.html#declaring-emitted-events

Reply:
I thought this would be the same since the properties can be used in the template as well without the return of the `defineProps`